src/Cms/ContentTranslation.php

```diff
@@ -177,7 +177,12 @@ protected function setCode(string $code)
      */
     protected function setContent(array $content = null)
     {
-        $this->content = $content;
+        if ($content !== null) {
+            $this->content = array_change_key_case((array)$content);
```

Comment:
Similarly here, if `$content` isn't `null`, it has to be already an array based on the parameter type hinting, right?

Reply:
In this case, yes!

Reply:
✅